spec/feature/posts_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'the post views:', type: :feature do
+  it 'renders the posts list view' do
+    visit '/users/1/posts'
+
+    expect(current_path).to eq('/users/1/posts')
+    expect(page).to have_text('User posts list')
+  end
+
+  it 'renders the post details view' do
+    visit '/users/1/posts/1'
+
+    expect(current_path).to eq('/users/1/posts/1')
+    expect(page).to have_text('User post details')
+  end
```

Comment:
- [ ] Kindly make sure to test (the response status / template rendered / body rendered) for each action, as per [this](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=If%20response%20status,correct%20placeholder%20text.) requirement.

Reply:
Hello @ShadyShawkat, kindly check the tests inside the spes/requests folder

Reply:
Hi @teckim, you tested for 2 of the required tests for each action. You tested for the response status and the template rendered, but not for the response body. I might have highlighted the wrong snippet of code.

Reply:
That's exactly what the code highlighted is doing!
`expect(page).to have_text('User posts list')`